pkg/domain/domain_sysvars.go

```diff
@@ -69,6 +69,15 @@ func (do *Domain) setPDClientDynamicOption(name, sVal string) {
 			break
 		}
 		variable.EnableTSOFollowerProxy.Store(val)
+	case variable.PDEnableFollowerHandleRegion:
+		val := variable.TiDBOptOn(sVal)
+		// Note: EnableFollowerHandle is only used for region API now.
```

Comment:
Should be EnableFollowerHandleRegion?

Reply:
As we discussed in [pr](https://github.com/tikv/pd/pull/7577), the variable names of pd client will not change for now. At present, we only need to define the names directly exposed to users in TiDB.

Reply:
Got it, I remember that.